src/Templates/Templates/CustomTypeEditor/.template.config/template.json

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
I haven't dug deep, but I have a nigling suspicion that the content here isn't quite correct (e.g., `"replaces": "net6.0-windows"` whereas this repo uses `net7.0-windows`, etc). 
I'm not quite sure where this file originated from, but I'd suggest considering the template.json file from dotnet/winforms repo.

Reply:
Exploring this right now. It's out-of-scope for the template solution review.
But yes, we would need to talk about this in our upcoming 1:1.